src/main/resources/init_db.sql

```diff
@@ -1,20 +1,39 @@
-CREATE SCHEMA IF NOT EXISTS `taxi` DEFAULT CHARACTER SET utf8;
-USE `taxi`;
+CREATE SCHEMA IF NOT EXISTS `taxi_db` DEFAULT CHARACTER SET utf8;
+USE `taxi_db`;
 
 DROP TABLE IF EXISTS `manufacturers`;
```

Comment:
this wont work if there is any data in ur tables as far as u will have rows referenced on ones from this table
same below